packages/web3-providers/src/explorer/index.ts

```diff
@@ -0,0 +1,35 @@
+import urlcat from 'urlcat'
+import { toTransaction } from './helpers'
+import type { Transaction } from './types'
+
+export async function getLatestTransactions(
+    account: string,
+    url: string,
+    {
+        offset = 10,
+        apikey,
+    }: Partial<{
+        offset?: number
+        apikey?: string
+    }> = {},
+) {
+    const response = await fetch(
+        urlcat(url, {
+            module: 'account',
+            action: 'txlist',
+            address: account.toLowerCase(),
+            startBlock: 0,
+            endblock: 999999999999,
```

Comment:
`await web3.eth.getBlockNumber()` use the latest block number

Reply:
Not necessary. The explorer API will handle this.